recipes/coin-cbc/all/conanfile.py

```diff
@@ -105,6 +108,10 @@ def _configure_autotools(self):
             "--without-blas"
             "--without-lapack"
         ]
+        if self.settings.os != "Windows":
+            configure_args = [
+                "--enable-cbc-parallel={}".format(yes_no(self.options.parallel))
```

Comment:
pthread should be added to system_libs in package_info() if Linux or FreeBSD

Reply:
done